x-pack/plugins/security_solution/common/api/entity_analytics/entity_store/common.schema.yaml

```diff
@@ -42,6 +42,49 @@ components:
         - updating
         - error
     
+    EngineComponentStatus:
```

Comment:
too many things called status 😅 
Each engine has a status field, then my upcoming PR also has a whole store Status thing. Now this one 🤔 
Can we come up with some different nomenclature? I'm thinking just `Health` for this?

Reply:
Hmm 🤔  
This is supposed to be a component-by-component breakdown of the Entity Descriptor status. Inside the components, there is already a `health` property, which is currently only used by the transform.
The status API also returns it, so I am still looking for a better name.  😓